### DISCUSSION TITLE
Remove `crossorigin="use-credentials"` from manifest link tag.

### DISCUSSION THREAD

**Discussion by umang-goyal** [vercel/next.js#65964] [Help]
### Summary

After migrating from Next.js 13 to 14 there was a change in manifest link tag with added attribute by default `crossorigin="use-credentials"`. 
After the addition of this attribute we see a CORS error being thrown when fetching manifest.json file.
Is there any way we can remove or set a different value for this attribute.

**Comment by cbhatia-birdeye** [comment]
Please someone help in this case. I am also getting the same issue after upgrading to Next v14.

**Comment by voskobovich** [comment]
Have the same issue 🫠

**Comment by jamesone** [comment]
Did you figure this one out? I am on 15 and am having the same issue.
